index.html

```diff
@@ -1108,7 +1108,7 @@
         <dfn>PaymentDetailsBase</dfn> dictionary
       </h2>
       <pre class="idl">
-        dictionary PaymentDetailsBase {
+        dictionary PaymentDetailsBase : Localizable {
```

Comment:
I want this to apply to `PaymentDetailsUpdate.error`, which is why  put it here.